test/bootstrap_form_test.rb

```diff
@@ -179,13 +181,17 @@ class BootstrapFormTest < ActionView::TestCase
             <input aria-required="true" required="required" class="form-control" id="user_email" name="user[email]" type="email" value="steve@example.com" />
           </div>
         </div>
-        <div class="form-check">
-          <input #{autocomplete_attr} name="user[terms]" type="hidden" value="0" />
-          <input class="form-check-input" id="user_terms" name="user[terms]" type="checkbox" value="1" />
-          <label class="form-check-label" for="user_terms">I agree to the terms</label>
+        <div class="mb-3 row">
+          <div class="col-sm-10 offset-sm-2">
+            <div class="form-check">
+              <input #{autocomplete_attr} name="user[terms]" type="hidden" value="0" />
+              <input class="form-check-input" id="user_terms" name="user[terms]" type="checkbox" value="1" />
+              <label class="form-check-label" for="user_terms">I agree to the terms</label>
+            </div>
+          </div>
         </div>
         <div class="mb-3 row">
-          <label class="form-label col-form-label col-sm-2" for="user_misc">Misc</label>
+          <label class="col-form-label pt-0 col-form-label col-sm-2" for="user_misc">Misc</label>
```

Comment:
Any idea why we expect `col-form-label` twice?

Reply:
Not sure.

Reply:
Looking more closely, I see that before we said, `form-label col-form-label`. [Current Bootstrap documentation](https://getbootstrap.com/docs/5.3/forms/layout/#horizontal-form-label-sizing) indicates we only need `col-form-label`, and only once (I could be wrong). Do you think you could clean that up so we only get `col-form-label` and only once?

Reply:
I will take a look 👍

Reply:
Reworked the code to only use the new label classes.